packages/app/src/runner/selector-playground/SelectorPlayground.cy.tsx

```diff
@@ -182,26 +160,20 @@ describe('SelectorPlayground', () => {
     })
   })
 
-  // TODO: fix this flaky test
-  it.skip('shows tooltips when buttons are focused', () => {
+  it('shows copy tooltip when button is focused', () => {
```

Comment:
Split these tests up. I think they're flaky because it's trying to do one tooltip after another which just doesn't work well with our tooltip components.